doc/nrf/app_dev/device_guides/thingy53/building_thingy53.rst

```diff
@@ -123,7 +123,7 @@ Complete the following steps to build and program using the |nRFVSC|:
 
 .. include:: /includes/vsc_build_and_run.txt
 
-3. Program the sample or application:
+1. Program the sample or application:
```

Comment:
Isn't this supposed to be step 3? 
![image](https://github.com/user-attachments/assets/0aa12967-c5bf-4a23-90ac-802495c3570c)